File: src/SensorValuesApp/app/src/main/java/reinhard/sensorvaluesapp/SensorValues.java
package reinhard.sensorvaluesapp;

import org.simpleframework.xml.Element;
import org.simpleframework.xml.Root;

import java.io.Serializable;

/**
 * Created by Bernhard on 25.05.2015.
 */

@Root
public class SensorValues {

    @Element
    private float AccelerometerX;

    @Element
    private float AccelerometerY;

    @Element
    private float AccelerometerZ;

    public float getAccelerometerX() {
        return AccelerometerX;
    }

    public void setAccelerometerX(float accelerometerX) {
        AccelerometerX = accelerometerX;
    }

    public float getAccelerometerY() {
        return AccelerometerY;
    }

    public void setAccelerometerY(float accelerometerY) {
        AccelerometerY = accelerometerY;
    }

    public float getAccelerometerZ() {
        return AccelerometerZ;
    }

    public void setAccelerometerZ(float accelerometerZ) {
        AccelerometerZ = accelerometerZ;
    }
}

File: src/SensorValueVisualization/SensorValues/SensorValues.cs
using System;

namespace SensorValues
{
    [Serializable]
    public class SensorValues
    {
        public double AccelerometerX { get; set; }

        public double AccelerometerY { get; set; }

        public double AccelerometerZ { get; set; }

        public override string ToString()
        {
            return String.Format("AccelerometerX: {0}, AccelerometerY: {1}, AccelerometerZ: {2}", AccelerometerX,
                AccelerometerY, AccelerometerZ);
        }
    }
}
